lib/containerize_task.rb

```diff
@@ -20,5 +20,11 @@ def containerize(description, dir)
     mapper = WorkloadMapper.new
     workloads = mapper.identify_workloads(description)
     mapper.write_compose_file(workloads, dir)
+    if workloads.empty?
+      Machinery::Ui.puts "No workloads detected."
+    else
+      Machinery::Ui.print "\nFound #{workloads.count} #{Machinery::pluralize(workloads.count, "workload")}."
```

Comment:
It should put out the list of workloads it found, not the count. This is much more useful information.